### DISCUSSION TITLE
Hide window in taskbar/alt-tab on Windows

### DISCUSSION THREAD

**Discussion by NotNite** [emilk/egui#1978] [Q&A]
Hi! I'm trying to make a Rainmeter-like application that sits on the back of my desktop with information about my machine, but my window (using eframe) is appearing in the alt-tab menu/taskbar.

I don't think there's a way to hide this in eframe - from some quick googling, I need to set `WS_EX_TOOLWINDOW` in the window style. Any ideas? Am I going to need to switch how I manage my window?

**Comment by averrin** [comment]
I'm looking for an answer too. I found something like `set_skip_taskbar` in `winit`, but I cannot figure out how to get a suitable window object. Any ideas?
